OpenRA.Mods.Common/Traits/Player/ClassicParallelProductionQueue.cs

```diff
@@ -192,6 +192,12 @@ protected override void BeginProduction(ProductionItem item, bool hasPriority)
 			base.BeginProduction(item, false);
 		}
 
+		protected override void PauseProduction(string itemName, bool paused)
+		{
+			foreach (var item in Queue.FindAll(a => a.Item == itemName))
+				item?.Pause(paused);
```

Comment:
Looks like the following would suffice? (for both overrides)

```c#
foreach (var item in Queue.Where(a => a.Item == itemName))
    item.Pause(paused);
```

Reply:
updated

Reply:
shouldn't it then also use `a?.Item` inside `FindAll`?